jekyll-assets/_includes/nav.html

```diff
@@ -9,8 +9,26 @@ <h3 class="contentshead" onclick="expandAndCollapse(this)">{{ item.title }}</h3>
       {% if item.sections[0] %}
         {% if page.url == item.path %}
         <div class="itemcontents" id="toc"></div>
+        <div class="itemcontents noJS">
```

Comment:
If JS is enabled, the first div here (the empty one) will be auto-filled by the tocify library. This new div is a backup only if JS is disabled.